modules/core/src/main/scala/org/tessellation/domain/aci/StateChannelOutput.scala

```diff
@@ -1,23 +1,9 @@
 package org.tessellation.domain.aci
 
-import cats.syntax.option._
-
-import org.tessellation.kernel.{Gistable, Ω}
 import org.tessellation.schema.address.Address
-
-import shapeless.Typeable
+import org.tessellation.security.signature.Signed
 
 case class StateChannelOutput(
```

Comment:
I think that the naming can be confusing. `Input` enclosed in `Output`. Maybe rename the `Input` to something similar to `StateChannelSnaphot` - since I guess it is a `Snapshot`.

Reply:
Yeah, I agree  I was a little bit confused too but didn't change it finally.